jekyll/_cci2/orbs-faq.md

```diff
@@ -9,164 +9,66 @@ order: 20
 
 This document describes various questions and technical issues that you may find helpful when working with orbs.
 
-* TOC
-{:toc}
+## Private orbs
 
-## Downloading, Integrating, and Testing Orbs
-{:.no_toc}
+* **Question:** Can orbs be made private?
 
-* Question: How do I download, integrate and test orbs?
+* **Answer:** Currently, all orbs published to the [Orb Registry]() are open source. Please subscribe to this feature request for updates: [Feature Request: private orbs](https://ideas.circleci.com/ideas/CCI-I-606)
 
-* Answer: You can import orbs using the `orbs` stanza in version 2.1 or higher of a CircleCI configuration. For example, if you want to publish an orb called  `hello-build` in the namespace `circleci` and have a published version `0.0.1`, import an orb like the example shown below:
+## Difference between commands and jobs
 
-```
-orbs:
-     hello: circleci/hello-build@0.0.1
-```
+* **Question:** What is the difference between commands and jobs?
 
-You may then invoke elements of the orb in your configuration by referencing elements under the key `hello`. For example, if an orb has a job called `hello-build`, you can invoke that job in a workflow like the example shown below.
+* **Answer:** Both [commands]({{site.baseurl}}/2.0/reusing-config/#the-commands-key) and [jobs]({{site.baseurl}}/2.0/reusing-config/#authoring-parameterized-jobs) are elements that can be used within orbs. _Commands_ contain one or many [steps]({{site.baseurl}}/2.0/configuration-reference/#steps), which contain the logic of the orb. Commands generally execute some shell code (bash). _Jobs_ are a definition of what steps/commands to run and within what [executor]({{site.baseurl}}/2.0/reusing-config/#executors) to run them. _Commands_ are invoked at the job level. _Jobs_ are invoked at the _[Workflow]({{site.baseurl}}/2.0/workflows/#workflows-configuration-examples)_ level.
```

Comment:
_Jobs_ are a definition of what steps/commands to run and within -a predefined [executor]- ?

"Commands invoked at the job level" and "Jobs are invoked at the workflow level" confuses me a bit. I guess I dont know what 'invoke' means here. 
Terrible suggestions: Orb commands are used in a project configs's job? Jobs are used in a project config's workflow? Commands are drop in replacements for steps in a job?
Also a link to jobs here: _Commands_ are invoked at the job level would be nice as well.

Reply:
Thanks for this feedback! I will think on how best to clear this up...

Reply:
I've tried to improve this now